README.md

```diff
@@ -31,9 +31,13 @@ Forward is the protocol used by Fluentd to route message between peers.
 | tls.key_file | "" | string | Used for mTLS. Path to the client TLS key to use |
 | shared_key | "" | string | A key string known by the server, used for authorization |
 | require_ack| false | bool | Protocol delivery acknowledgment for log streams : true = at-least-once, false = at-most-once |
+| skip_fail_on_invalid_tcp_endpoint | false | bool | Controls whether to fail if the endpoint is invalid. This is useful for cases where the collector is started before the endpoint becomes available |
```

Comment:
Nice enhancement. We could also fail each time like it is done on the `carbonexporter` https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/carbonexporter/config.go#L39

Continuing in that sense to not break the actual behavior, it is better to put a default value of `true`. Because intentionally I prefer to skip a failure if the tcp endpoint is invalid ; we need the opentelemetry-collector to start and log it can't connect and auto reconnect for new logs, but I have another proposition below.

What do you think of using a struct like:
```yaml
fluentforward/allsettings:
  endpoint: 
    tcp_addr: "localhost:24224"
    validate_tcp_resolution: true
```

With `validate_tcp_resolution: false` by default ?